src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto , sans-serif;
+  font-size: 12px;
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md), the 3rd item